lib/scala/docs-generator/src/main/scala/org/enso/DocsGeneratorMain.scala

```diff
@@ -0,0 +1,124 @@
+package org.enso
+
+import java.io._
+import scala.util.Using
+import scala.io.Source
+import scalatags.Text.{all => HTML}
+import HTML._
+import docsgenerator.DocsGenerator._
+import docsgenerator.TreeOfCommonPrefixes._
+
+/** The Docs Generator script.
+  * Used to create HTML documentation for Enso Standard Library, and also JS
+  * files with react components for the "component reference" website.
+  */
+object DocsGeneratorMain extends App {
+
+  /// Files
+  val path = "./distribution/std-lib/Standard/src"
+  val allFiles = traverse(new File(path)).filter(f =>
+    f.isFile && f.getName.endsWith(".enso")
+  )
+  val allFileNames = allFiles.map(
+    _.getPath
+      .replace(path + "/", "")
+      .replace(".enso", "")
+  )
+
+  /// HTML's w/o react & tree (for IDE)
+  val allPrograms = allFiles
+    .map(f => Using(Source.fromFile(f, "UTF-8")) { s => s.mkString })
+    .toList
+  val allDocs = allPrograms
+    .map(s => generate(s.getOrElse("")))
+    .map(mapIfEmpty)
+    .map(removeUnnecessaryDivs)
+  val allDocFiles = allFiles.map(
+    _.getPath
+      .replace(".enso", ".html")
+      .replace("Standard/src", "docs")
+  )
```

Comment:
Do keep in mind that we don't actually want this rendered HTML. From the engine perspective we care about the chunks so we can assemble actually useful blocks of documentation that include extension methods, re-exports, and so on.